### ISSUE TITLE
[Docs] Example for components > navbar > external content is ... odd/contrived

### ISSUE THREAD

**Issue by patrickhlauke** [twbs/bootstrap#32041]
https://v5.getbootstrap.com/docs/5.0/components/navbar/#external-content is a strangely convoluted/contrived example - i'm failing to come up with an actual good use case for the feature in the first place, but what's shown there in the example seems bizarre. can we come up with something closer to a real-life situation where this might be of use?

**Comment by patrickhlauke** [comment]
the example, as it stands, also has weird reading order/focus order issues. if the expanded content had a focusable element, a keyboard user would activate the menu button and then have to navigate in reverse (`SHIFT + TAB`) to get through the revealed stuff. same with screen reader user using reading keys ... they open the menu, then have to cursor backwards to read the stuff (in reverse)

**Comment by Wolfr** [comment]
I think the example is legit for the use case that your mobile menu pops up a separate div.
